sympy/core/power.py

```diff
@@ -1575,7 +1578,10 @@ def mul(d1, d2):
             tk = mul(tk, gterms)
             k += S.One
 
-        inco, inex = coeff_exp(f**e, x)
+        if not e.is_integer and m.is_zero and f.is_real and f.is_negative and im((b - f).dir(x, cdir)) < 0:
```

Comment:
This is a long(ish) line.

Reply:
Will reduce its length using parenthesis.